rules.mk

```diff
@@ -193,7 +197,7 @@ LDFLAGS.release +=
 
 COMMON_FLAGS.debug += -O0
 COMMON_FLAGS.release += -O3
-COMMON_FLAGS += -ggdb -march=native -Wall -Wextra -Wno-attributes -Wno-unused-parameter -pthread $(COMMON_FLAGS.$(BLD))
+COMMON_FLAGS += -ggdb -mtune=native -Wall -Wextra -Wno-attributes -Wno-unused-parameter -pthread $(COMMON_FLAGS.$(BLD))
```

Comment:
There is no [`-march` on Power ](https://gcc.gnu.org/onlinedocs/gcc/RS_002f6000-and-PowerPC-Options.html), but I'm using `-mcpu` now and reverted the Intel version.